src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -682,6 +695,18 @@ private static bool TryGetIntQueryParameterValue(NameValueCollection queryString
             return int.TryParse(value, out intValue);
         }
 
+        private static bool TryGetTimeSpanQueryParameterValue(NameValueCollection queryStringNameValueCollection, string queryParameterName, out TimeSpan? timeSpanValue)
+        {
+            timeSpanValue = null;
+            string value = queryStringNameValueCollection[queryParameterName];
+            if (value != null)
+            {
+                timeSpanValue = TimeSpan.FromSeconds(double.Parse(value));

```

Comment:
We should probably handle the case where they provide a non-numerical value for the timeout/polling interval and return a 400 instead of a 500. I think that needs to be handled in other places throughout this class, so it may be worth filing a separate issue to tackle later.

Reply:
I see. In the other query parameter value methods we just ignore incorrect input. For now I should at least fix this method to do the same.